package.json

```diff
@@ -69,6 +70,8 @@
     "redux-mock-store": "0.0.6",
     "salinity": "0.0.8",
     "style-loader": "0.13.0",
-    "webpack": "1.12.12"
+    "webpack": "1.12.12",
+    "webpack-dev-server": "^1.14.1",
```

Comment:
Please lock down these dependencies - we don't use `^`. (You can do `npm install --save-exact` in future to avoid having to remember to change in the package.json.)